### PR TITLE
Proposal to enable cWETHv3 market on Goerli

### PR DESCRIPTION
**Pull request by kevincheng96**
Enact run: https://github.com/compound-finance/comet/actions/runs/3849939037/jobs/6559516200
Proposal executed: https://goerli.etherscan.io/tx/0xc7e1755a28378570de1c07b69736ec57e4da69da22f0a9e8cf692fddb551c043

### CODE REVIEW

File: deployments/goerli/weth/configuration.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   },
   "tracking": {
     "indexScale": "1e15",
-    "baseSupplySpeed": "0e15",
+    "baseSupplySpeed": "447916666666e0",
```
Comment (kevincheng96) [inline deployments/goerli/weth/configuration.json:23]: I could modify the `stringToBigInt` to accept non-scientific notation numbers as well. But this is a quick workaround.

**Review by jflatow — APPROVED** [review]
🧼

**Comment by kevincheng96** [comment]
Enact run: https://github.com/compound-finance/comet/actions/runs/3849939037/jobs/6559516200

**Comment by kevincheng96** [comment]
Proposal executed: https://goerli.etherscan.io/tx/0xc7e1755a28378570de1c07b69736ec57e4da69da22f0a9e8cf692fddb551c043

**Pull request merged**